contracts/transmuter/libraries/LibOracle.sol

```diff
@@ -197,6 +187,9 @@ library LibOracle {
         } else if (readType == OracleReadType.MAX) {
             (uint256 maxValue, , , ) = abi.decode(data, (uint256, uint96, uint96, uint32));
             return maxValue;
+        } else if (readType == OracleReadType.MORPHO_ORACLE) {
```

Comment:
This makes external call while we could get the same thing with ERC4626 rate + chainlink or Pyth rates. 
I see that Morpho implementation is nice because it allows to combine chainlink like feed + an ERC4626 (knowing that we don't need the quote part as we will always stop on the transmuter at fiat). But you can already with our implementation with the `quoteAmount` part by just adding a type `ERC4626`